themes/reactos/layouts/partials/shared-list.html

```diff
@@ -17,7 +17,7 @@
 						{{ range $.paginator.Pages }}
 						<section class="post">
 							<div class="row">
-								<div class="col-md-3">
+								<div class="col-sm-3 hidden-xs">
```

Comment:
This does not affect the images in the news list on the home page, is that intentional?

Reply:
Nope, I forgot that